client/lib/products-values/constants.js

```diff
@@ -69,6 +69,24 @@ export const JETPACK_PRODUCT_DISPLAY_NAMES = {
 	...JETPACK_BACKUP_PRODUCT_DISPLAY_NAMES,
 };
 
+export const JETPACK_BACKUP_PRODUCT_DAILY_TAGLINE = translate(
```

Comment:
Sure, I can do that 👍

Reply:
I'm going to keep it as is for now because we've got more constant names like this (e.g. `JETPACK_BACKUP_PRODUCT_DAILY_DISPLAY_NAME` or `PRODUCT_JETPACK_BACKUP_REALTIME_DESCRIPTION`). I don't want to "pollute" this PR with more changes.